converter/src/test/java/gov/cms/qpp/acceptance/cpc/CPCAcceptanceFixture.java

```diff
@@ -0,0 +1,24 @@
+package gov.cms.qpp.acceptance.cpc;
+
+import java.util.List;
+
+public class CPCAcceptanceFixture {
```

Comment:
We don't normally javadoc test classes, but could you put a bit of documentation here to explain a bit about what this class does?

Reply:
kk